round4/Ideation.md

```diff
@@ -0,0 +1,30 @@
+
+## Target Audience
+The target audience for our Chrome Extension includes frequent travelers, expatriates, international students, online shoppers buying from foreign websites, and professionals dealing with cross-border transactions. This audience primarily consists of individuals who regularly require quick and reliable currency conversion to budget, plan, and make informed decisions related to their international activities.
+
+## Description
+Our Chrome Extension offers a quick and easy way to convert currencies right from the browser toolbar. Users can select their base and target currencies, input the amount they wish to convert, and receive real-time conversion rates. The extension also features a customizable watchlist for monitoring favorite currencies and historical rate charts for making informed decisions.
+
+## Selling Points
+- Real-time currency conversion for over 150 currencies, ensuring users have access to the most current rates.
+- A customizable watchlist that allows users to keep track of and quickly access their most-used currencies.
+- Historical data charts for each currency pair, helping users analyze trends and make more informed decisions.
+- A simple, user-friendly interface that enables quick conversions without navigating away from the current page.
+- Automatic detection of currencies on web pages and the option to convert displayed prices with a single click.
+## User Stories
+- As a frequent traveler, I want to quickly convert expenses into my home currency so that I can manage my budget more effectively.
+- As an online shopper, I want to see prices in my local currency on international websites so that I can understand how much I will be spending without having to manually convert the prices.
+- As an expatriate, I want to monitor the exchange rate between my home currency and the local currency so that I can send money home when rates are favorable.
+- As a forex trader, I want to access historical exchange rates directly from my browser so that I can make informed trading decisions based on trends.
+- As a student studying abroad, I want to convert tuition and living expenses into my parents' currency so they can easily understand the costs involved.
+## Notes
+A potential challenge might be ensuring that the currency conversion rates are updated in real-time, requiring a reliable and cost-effective API service. Additionally, we want to ensure that our extension does not significantly impact the browser's performance, especially when monitoring multiple currencies. User privacy is paramount, so we plan to design the extension to operate without collecting sensitive user data.
```

Comment:
I like that you are thinking about the challenges that might occur with this extension. Updating the currency in real time might be the most difficult part but there should be a way that will easily fetch the data from a source (maybe web scraping) to help you accomplish the task. I also do not think there is a need to collect user data because it is just currency conversion